Samples/kmp-app/androidApp/build.gradle.kts

```diff
@@ -0,0 +1,27 @@
+plugins {
+    id("com.android.application")
+    kotlin("android")
+}
+
+android {
+    compileSdk = 32
+    defaultConfig {
+        applicationId = "sample.kpm_app.android"
+        minSdk = 30
+        targetSdk = 32
+        versionCode = 1
+        versionName = "1.0"
+    }
+    buildTypes {
+        getByName("release") {
+            isMinifyEnabled = false
+        }
+    }
+}
+
+dependencies {
+    implementation(project(":shared"))
```

Comment:
Feels like our SDK is called `shared`, but it should be something similar to `sentry-kotlin-multiplatform`

Reply:
The shared module deals with dividing common, iOS and Android code so it contains the dependency in shared/build.gradle.kts,